src/protect-api/appsettings.json

```
@@ -2,7 +2,8 @@
   "AzureAd": {
     "Instance": "https://login.microsoftonline.com/",
     "ClientId": "[Enter_your_application_client_ID_from_Azure_Portal, _e.g._2ec40e65-ba09-4853-bcde-bcb60029e596]",
-    "TenantId": "[Enter_your_azure_ad_tenant_ID_from_Azure_Portal]"
+    "TenantId": "[Enter_your_azure_ad_tenant_ID_from_Azure_Portal]",
+    "Scopes": "forescast.read"

```

```suggestion
    "Scopes": "Forecast.Read"
```
Can we make all places (including the readme instructions have it as `Forecast.Read`)?

---

thanks for pointing this out @ckittel 

done | Addressed from [e9a99](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/12/commits/e9a99b3f3d7fa630f93521656d07bb97ba6ef534)

---

Thanks!